homeassistant/components/vilfo/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/vilfo",
   "iot_class": "local_polling",
   "loggers": ["vilfo"],
-  "requirements": ["vilfo-api-client==0.3.2"]
-}
+  "requirements": ["vilfo-api-client==0.4.1"]
```

Comment:
It would be good if someone can test the vilfo integration with this change.

Reply:
Sorry for the late reply. I'll have a look at this within the next couple of days and if anything needs to be updated in the Vilfo integration, I'll try to do that as well 🙂

Reply:
Great, thanks!

Reply:
Hi @ManneW just wondering do you think you might have some time to confirm the uplift of the version of your package looks OK to merge.